akka-docs/src/main/paradox/stream/operators/Flow/lazyInitAsync.md

```diff
@@ -1,6 +1,6 @@
 # Flow.lazyInitAsync
 
-Creates a real `Flow` upon receiving the first element by calling relevant `flowFactory` given as an argument.
+`fromCompletionStage` has been deprecated in 2.6.0 use `Flow.lazyFutureFlow` instead.
```

Comment:
Wanted to link to the right document here but ref must be relative and this is injected in the operator-toc one directory down so relative bath becomes the wrong path if I do.